web/models/meeting.ex

```diff
@@ -8,32 +9,43 @@ defmodule Meetings.Meeting do
   
   def new(data) do
     %Meetings.Meeting{ 
-      title: elem(data, 0),
-      location: elem(data, 1),
+      id: elem(data, 0),
+      title: elem(data, 1),
+      location: elem(data, 2),
       date: %DateTime{calendar: Calendar.ISO, time_zone: "CST/UTC", zone_abbr: "CST", std_offset: 6, utc_offset: 6,
-      year: elem(data, 2), month: elem(data, 3), day: elem(data, 4), hour: elem(data, 5), minute: elem(data, 6), second: 0 },
-      duration: elem(data, 7),
+      year: elem(data, 3), month: elem(data, 4), day: elem(data, 5), hour: elem(data, 6), minute: elem(data, 7), second: 0 },
```

Comment:
Being picky on formatting here, but any way this could be cleaned up so the keys for the `DateTime` struct are indented or something so it's clear what's a part of that and what's part of the `Meeting` struct?